### DISCUSSION TITLE
Slider filled?

### DISCUSSION THREAD

**Discussion by NeoCoderMatrix86** [Megabit/Blazorise#5898] [Q&A]
Hello,

I use a slider during an audio playback in my application (like in VLC). Is it possible to fill the left bar till the current position?

![grafik](https://github.com/user-attachments/assets/b3024838-5b3e-4a7e-b954-8585ae7671a0)

**Comment by stsrki** [comment]
You can use CSS to adjust the fill color. Try this https://stackoverflow.com/questions/18389224/how-to-style-html5-range-input-to-have-different-color-before-and-after-slider

**Reply by NeoCoderMatrix86** [reply]
Thanks, I'll have a look and try to customize the css. Maybe a setting like "Filled" for slider would be a good enhancement :).
